CFC_WebApp/api/cfc_webapp.py

```diff
@@ -352,6 +354,10 @@ def javascriptCallback(clientName, method):
   client.callJavascriptCallback(client_key, method, request.params)
   return {'status': 'ok'}
 
+@get('/fbregister')
```

Comment:
This is not really used anywhere in the current implementation.
Remove until it is actually needed.